src/pages/scripting/how-it-works/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How it works
+description: More details about UXP Scripting
+contributors:
+  - https://github.com/amandahuarng
+---
+
+# How UXP Scripting Works
+Each Photoshop script file is a plain text file with a `.psjs` file extension. We plan on expanding UXP scripting to other applications and supporting opening scripts via double-click in the future. When this happens, the file extension will help identify which application the script should be launched in. 
+
+## Execution Context
+Photoshop sets an execution context while invoking a script.
+
+Within each execution context, only one script can be executed at a time. You cannot invoke another script from the running script. Using the UXP script module, you can access `ExecutionContext`.
```

Comment:
AFAIK there will only be one execution context. Can you please remove 'each' from  _Within each execution context, only one script can be executed at a time_